### PR TITLE
Migrate to openai 1.0.0

### PR DESCRIPTION
**Pull request by picklelo**

### CODE REVIEW

File: webui/webui/state.py [hunk 1]
```diff
@@ -1,14 +1,25 @@
 import os
 import requests
 import json
-import openai
+from openai import OpenAI
 import reflex as rx
 
+<<<<<<< Updated upstream
```
Comment (Huge) [inline webui/webui/state.py]: this is a bug, bro

File: webui/webui/state.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
 openai.api_key = os.environ["OPENAI_API_KEY"]
 openai.api_base = os.getenv("OPENAI_API_BASE", "https://api.openai.com/v1")
 
 BAIDU_API_KEY = os.environ["BAIDU_API_KEY"]
 BAIDU_SECRET_KEY = os.environ["BAIDU_SECRET_KEY"]
+=======
```
Comment (Huge) [inline webui/webui/state.py]: this fails

File: webui/webui/state.py [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+client = OpenAI(api_key=os.getenv("OPENAIa_API_KEY"))
+
+BAIDU_API_KEY = os.getenv("BAIDU_API_KEY")
+BAIDU_SECRET_KEY = os.getenv("BAIDU_SECRET_KEY")
+
+
+if not client.api_key and not BAIDU_API_KEY:
+    raise Exception("Please set OPENAI_API_KEY or BAIDU_API_KEY")
+>>>>>>> Stashed changes
```
Comment (Huge) [inline webui/webui/state.py]: check your stash merging

**Review by Huge — COMMENTED** [review]
bad git artifacts

File: webui/webui/state.py [hunk 4]
```diff
@@ -1,14 +1,25 @@
 import os
 import requests
 import json
-import openai
+from openai import OpenAI
 import reflex as rx
 
+<<<<<<< Updated upstream
 openai.api_key = os.environ["OPENAI_API_KEY"]
```
Comment (Huge) [inline webui/webui/state.py]: this won't work now,
it is likely better to be removed.?.

**Review by Huge — CHANGES_REQUESTED** [review]

**Comment by picklelo** [comment]
@Huge thanks for the comments, I made some updates, can you try it out and see if it works for you now

**Pull request closed without merging**